src/main/java/io/jsonwebtoken/impl/ValidationHelper.java

```diff
@@ -0,0 +1,193 @@
+package io.jsonwebtoken.impl;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Clock;
+import io.jsonwebtoken.ExpiredJwtException;
+import io.jsonwebtoken.Header;
+import io.jsonwebtoken.PrematureJwtException;
+
+/**
+ * This class contains static methods for validation of JWT claims.
+ */
+public class ValidationHelper {
```

Comment:
The problem with Validator interface, that we cannot put additional method parameters into the validate method. For example for checking expires claim we need to optionally provide allowedClockSkewMillis or clock. Do you have any suggestion for this?